src/angularjs/src/app/users/detail/users-detail.html

```diff
@@ -45,7 +45,7 @@ <h1 ng-if="user.editing">Edit User</h1>
             <div class="form-group">
```

Comment:
Now that the logic here to disable this control is getting a bit complicated, would it be clearer to just `ng-if` this block into existence if `user.isAdminUser || user.isOrgAdminUser || user.loggedInEmail === user.user.email`?